### PR TITLE
Add a new podAction: stopHost

### PR DESCRIPTION
**Pull request by seeker89**
## TL;DR

Often, you want to stop a host that runs a particular pod. Right now, you need to do some gymnastics to achieve that.
With `stopHost`, you can select all the pods you like, and stop all the hosts that are involved in running that set.

## Demo

This will restart the nodes at the end of the scenario

```yaml
scenarios:
- name: Hello chaos!
  steps:
  - podAction:
      matches:
        - namespace: something
      filters:
        - randomSample:
            size: 1
      actions:
        - stopHost:
            autoRestart: true
  - wait:
      seconds: 30
```

### CODE REVIEW

File: powerfulseal/policy/action_nodes_pods.py [hunk 1]
```diff
@@ -208,6 +209,7 @@ def action_wait(self, items, params):
         sleep_time = params.get("seconds", 0)
         self.logger.info("Action sleep for %s seconds", sleep_time)
         time.sleep(sleep_time)
+        return True
```
Comment (flyingmelon123) [inline powerfulseal/policy/action_nodes_pods.py:212]: curious what does `action_wait` need a return value?

File: powerfulseal/policy/action_pods.py [hunk 2]
```diff
@@ -16,9 +16,26 @@
...
+from powerfulseal import makeLogger
 from powerfulseal.metriccollectors.collector import POD_SOURCE
 from .action_nodes_pods import ActionNodesPods
 
+class StartHostAction():
+    """ A little helper class to start hosts in cleanup """
+    def __init__(self, driver, host, logger=None):
+        self.driver = driver
+        self.host = host
+        self.logger = logger
```
Comment (flyingmelon123) [inline powerfulseal/policy/action_pods.py:28]: if logger is `None` dont we want to make a logger?

File: powerfulseal/policy/action_pods.py [hunk 3]
```diff
@@ -154,3 +172,27 @@ def action_check_pod_state(self, pods, params):
...
+    def action_stop_host(self, pods, params):
+        """ Action to stop a node.
+        """
+        self.inventory.sync()
+        host_ips = list(set([p.host_ip for p in pods]))
+        for host_ip in host_ips:
+            host = self.inventory.get_node_by_ip(host_ip)
+            if host is None:
+                self.logger.warning("Couldn't find host with IP %r", host_ip)
+                return False
```
Comment (flyingmelon123) [inline powerfulseal/policy/action_pods.py:185]: so we do not continue once running into a bad ip?

File: powerfulseal/policy/action_nodes_pods.py (hunk 1 above)
Comment (seeker89) [reply powerfulseal/policy/action_nodes_pods.py:212]: Every action needs a return value to signal whether it succeeded or not.

File: powerfulseal/policy/action_pods.py (hunk 2 above)
Comment (seeker89) [reply powerfulseal/policy/action_pods.py]: Good catch, thanks!

File: powerfulseal/policy/action_pods.py (hunk 3 above)
Comment (seeker89) [reply powerfulseal/policy/action_pods.py:185]: That's the idea.

**Pull request merged**